### PR TITLE
Suggest checking line-endings on ENOENT ("no such file or directory")

### PR DESCRIPTION
**Pull request by briandealwis**
Detect ENOENT and suggest line endings.

```[builder] Running "./mvnw clean package --batch-mode -DskipTests -Dhttp.keepAlive=false --quiet"```
```[builder] Done "./mvnw clean package --batch-mode -DskipTests -Dhttp.keepAli..." (2.314324ms)```
```[builder] Failure: (ID: 7d0b3ca3) executing command "./mvnw clean package --batch-mode -DskipTests -> Dhttp.keepAlive=false --quiet": fork/exec ./mvnw: no such file or directory: ensure script does not have CR-LF line endings```

### CODE REVIEW

File: pkg/gcpbuildpack/BUILD.bazel [hunk 1]
```diff
@@ -24,6 +24,7 @@ go_library(
     deps = [
         "//pkg/env",
         "@com_github_buildpacks_libcnb//:go_default_library",
+        "@org_golang_x_sys//unix:go_default_library",
```
[the remark this answers is not in the document]
Comment (briandealwis) [reply pkg/gcpbuildpack/BUILD.bazel:27]: I kinda wanted to see if it would break :-)

[the remark this answers is not in the document]
Comment (lukasberger) [reply pkg/gcpbuildpack/BUILD.bazel:27]: Let's see.

**Comment by briandealwis** [comment]
Needed to add a go.sum (generated by `go mod tidy`) as the license check was failing:
```
Step #1 - "verify-licenses": Error: go [list -e -json -compiled=false -test=false -export=false -deps=true -find=false -- ./...]: exit status 1: go: github.com/BurntSushi/toml@v0.3.1: missing go.sum entry; to add it:
Step #1 - "verify-licenses": 	go mod download github.com/BurntSushi/toml
```

A go.sum seems a good idea.

**Comment by briandealwis** [comment]
Fixed by a00353abdf1e33a47013c94e14b0c3f2b3148033

**Pull request closed without merging**